test/user_auth_test.go

```diff
@@ -390,21 +381,42 @@ func TestGrantUserCollection(t *testing.T) {
 	if _, err := authCol.ReadDocument(nil, meta1.Key, &doc); !driver.IsForbidden(err) {
 		t.Errorf("Expected failure, got %s", describe(err))
 	}
-
 	// Now remove explicit collection access
 	if err := u.RemoveCollectionAccess(nil, col); err != nil {
 		t.Fatalf("RemoveCollectionAccess failed: %s", describe(err))
 	}
+	expected := driver.GrantNone
```

Comment:
What is this?
After version 3.3.4 we have an (behavioural) API change?

Reply:
Yes 3.3.4 changes some behaviour, it mentioned on the CHANGELOG.